e2e/storybook-angular/src/storybook-angular.test.ts

```diff
@@ -55,7 +55,7 @@ describe('Storybook for Angular', () => {
       `generate @nrwl/angular:storybook-configuration ${angularStorybookLib} --generateStories --no-interactive`
     );
 
-    checkFilesExist(`.storybook/main.js`);
+    checkFilesExist(`libs/${angularStorybookLib}/.storybook/main.js`);
     writeFileSync(
       tmpProjPath(`.storybook/main.js`),
```

Comment:
Not sure if this should remain as is, or should be also changed to `libs/${angularStorybookLib}/.storybook/main.js`

The same goes for the check on line 80

cc @mandarini